testing/chain_test.go

```diff
@@ -23,11 +23,11 @@ func TestChangeValSet(t *testing.T) {
 	amount2, ok := sdk.NewIntFromString("30000000000000000000")
 	require.True(t, ok)
 
-	val := chainA.App.GetStakingKeeper().GetValidators(chainA.GetContext(), 4)
+	val := chainA.App.GetIBCTestStakingKeeper().GetValidators(chainA.GetContext(), 4)
 
-	chainA.App.GetStakingKeeper().Delegate(chainA.GetContext(), chainA.SenderAccounts[1].SenderAccount.GetAddress(),
+	chainA.App.GetIBCTestStakingKeeper().Delegate(chainA.GetContext(), chainA.SenderAccounts[1].SenderAccount.GetAddress(),
 		amount, types.Unbonded, val[1], true)
-	chainA.App.GetStakingKeeper().Delegate(chainA.GetContext(), chainA.SenderAccounts[3].SenderAccount.GetAddress(),
+	chainA.App.GetIBCTestStakingKeeper().Delegate(chainA.GetContext(), chainA.SenderAccounts[3].SenderAccount.GetAddress(),
```

Comment:
Since this is the test file, maybe we can use `chainA.GetSimApp().StakingKeeper`